mappings/on_cancel.jsonata

```diff
@@ -42,6 +42,7 @@
                 "payments": message.order.payments,
                 "cancellation_terms": message.order.cancellation_terms
             }
-        }
+        },
+        "rawResponse": $includeRawResponse ? $ :{}
```

Comment:
instead of creating response like this
`"rawResponse":{"responses":[{"context":{},"message":{}}...]}`
i have created a response like this 
`"rawResponse":{"context":{},"message":{}}`
coz jsonata mapping is done on each element of the responses array
same implemented on other places also
cc: @vbabuEM

Reply:
What will happen in search request when there are more than one response?